src/GitHub.UI/Assets/TextBlocks.xaml

```diff
@@ -71,7 +71,7 @@
                             </Border>
                         </Border>
 
-                        <Grid Margin="1,0,0,0">
+                        <Grid Margin="1,2,0,0">
```

Comment:
Does this affect the UI in the Login/RepositoryCreation/RepositoryClone controls? Are they using this as well?